systemtest/src/test/java/io/strimzi/systemtest/topic/TopicScalabilityST.java

```diff
@@ -31,6 +32,13 @@ void testBigAmountOfTopicsCreatingViaK8s() {
                 currentTopic, 3, 1, 1).build());
         }
 
+        for (int i = 0; i < NUMBER_OF_TOPICS; i = i + SAMPLE_OFFSET) {
+            String currentTopic = topicName + i;
+            LOGGER.debug("Verifying that {} topic CR has Ready status", currentTopic);
+
+            KafkaTopicUtils.waitForKafkaTopicStatus(currentTopic, "Ready");
+        }
+
```

Comment:
Do we know how long this actually takes? The fact that you're having to change this suggests that the TO isn't really scaling very well the large numbers of topics, which is a problem.

Reply:
1000 TOPICS -> ~ 13 mins. The problem was that I was creating 2500 topics and `do not` wait on `Ready` status like in this change.

Reply:
I wonder how quickly you can create 1000 topics in the same cluster without using the TO. Is there any chance you could find out? My feeling is that the ZK writes which TO does will slow it down, but IDK the difference as a factor. If it was 2× slower that might be OK. If it was 10× slower I think we'd definitely need to look into it more. 

Another thing is how long is takes to do periodic reconciliation when there are 1000 topics. It shouldn't really affect the (eventual) correctness, but again it would be good to get a feel for these numbers.

Reply:
I have tried to create 1000 topics via AdminClient and it was in approximately ~9 mins.

Reply:
@see-quick thanks for that, it's a useful datapoint.